client/lib/react-player/bower.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Do we need bower.json ? 
If I understand it correctly I see there is only dependency in this which is React which we are already adding in the index.html as a CDN